filter/kubernetes_filter.go

```diff
@@ -107,7 +108,12 @@ func (filter *KubernetesContainerFilter) getWorkspacePods() (*v1.PodList, error)
 		return nil, errors.New("unable to get current workspace id")
 	}
 
-	filterOptions := metav1.ListOptions{LabelSelector: WsIdLabel + "=" + workspaceID, FieldSelector: "status.phase=Running"}
+	targetLabelSelector := os.Getenv(TargetPodSelectorEnvVar)
+	if WsIdLabel == "" {
+		targetLabelSelector = WsIdLabel + "=" + workspaceID
+	}
```

Comment:
I'm not sure I understand the logic here -- `WsIdLabel` is never `""` since it's a const.

Reply:
Oops yeah that was leftover from before I updated the PR. I'll fix it